client/src/main/java/org/spine3/base/Commands.java

```diff
@@ -70,17 +71,21 @@ public static CommandId generateId() {
     }
 
     /**
-     * Creates new command context with the current time
+     * Creates a new command context with the current time.
+     *
      * @param userId the actor id
      * @param zoneOffset the offset of the timezone in which the user works
+     * @param boundedContextName the name of the current context
+     * @see CommandFactory
      */
-    public static CommandContext createContext(UserId userId, ZoneOffset zoneOffset) {
+    public static CommandContext createContext(UserId userId, ZoneOffset zoneOffset, String boundedContextName) {
```

Comment:
The last parameter will be used _only_ for integration events. We should not modify the core API for such a minority of cases. 

It would be better to modify the set of fields of `IntegrationEvent` eliminating the `context` field, which has type `EventContext`. It would solve the problem of necessity of having a Bounded Context name.